### ISSUE TITLE
Set-up SOLR in Prod

### ISSUE THREAD

**Issue by LJTrent** [bcgov/name-examination#865]
##### _**Detailed Description**_

Set up all SOLR pods/deployments in prod including the admin app.

##### _**Definition of Done**_ (:one:-Mandatory to add to the Backlog, :two:-Mandatory to add to the Sprint Backlog)
- [ ] Acceptance Criteria Defined :one:
- [ ] Estimate :two:
- [ ] Priority Label :one:
- [ ] Task Label :one:
- [ ] Assignee :two:
- [ ] Sprint Goal (in line with the goal of the sprint) :two:
----
----

**Comment by WalterMoar** [comment]
postgresql-solr, solr, and solr-feeder are set up in production. solr-admin-app needs some work so that people can't edit in production - this work needs to be done before it is deployed, but solr-admin-add is not needed for production functionality.

The trademarks core is currently being loaded, it is the most time consuming. After the Oracle database changes are made on the 18th we can load the other two cores (~30 minutes).

**Comment by WalterMoar** [comment]
Trademarks core loading is complete. As with the dev load, no problems with Solr restarting - unsure why test had so many problems.

**Comment by WalterMoar** [comment]
Requirements in the description are met, although the names and possible.conflicts cores cannot be loaded until the database changes are made on the 18th.
